flaml/nlp/utils.py

```diff
@@ -358,7 +437,7 @@ class HPOArgs:
         metadata={"help": "model path model for HPO"},
     )
 
-    fp16: bool = field(default=True, metadata={"help": "whether to use the FP16 mode"})
+    fp16: bool = field(default=False, metadata={"help": "whether to use the FP16 mode"})
```

Comment:
This seems a big change. @liususan091219 could you confirm this is desired?

Reply:
This is on purpose. If we set it to True all the test should explicitly specify fp16. There are a lot of tests so I decided the default should be false.

Reply:
```suggestion
    fp16: bool = field(default=True, metadata={"help": "whether to use the FP16 mode"})
```